### ISSUE TITLE
Possible Fog of War Bug

### ISSUE THREAD

**Issue by coolyoshi** [npruehs/ue4-rts#158]
Not sure if it's just me but player 1 sometimes can't initialize fog of war properly.
The screen might go entirely black, or partially black (covered partially by fog), but when it breaks the result is always that fog just doesn't work.

**Comment by npruehs** [comment]
Hey @coolyoshi, have you been able to solve this?

**Comment by coolyoshi** [comment]
I think this bug might pop up if the gamestate has too much setup logic/too slow. Did a workaround by initializing vision after a delay once all players are loaded in. It's not a proper solution though.

**Comment by LegendsofBarcraft** [comment]
Im having the same issue. What was your exact solution Coolyoshi if I may ask?

**Comment by coolyoshi** [comment]
I wouldn't call it a solution but I just wait till all players loaded into game. Then after that I spawn the actors into the game after a 7 second delay. Basically that delay allows enough time to set everything up and avoid fog issues I guess

**Comment by LegendsofBarcraft** [comment]
Hmmm interesting, and you add the delay into the gamestate?

**Comment by coolyoshi** [comment]
Yeah. Actually I'm not really sure if it fixes it 100%, just seems to improve it alot atleast for me.

**Comment by npruehs** [comment]
Can you share more details about your setup? Listen server or dedicated? Example map of the plugin or your own one?

**Comment by LegendsofBarcraft** [comment]
Hi Nick, I just added a landscape on the example map. Pablo was trying to replicate the issue, but couldnt do it.  

EDIT: Ooops, wrong thread, but still relevant I guess?

**Comment by LegendsofBarcraft** [comment]
I can show it to you on the discord server :)

**Comment by LegendsofBarcraft** [comment]
I will be there for another 5 hours cca.

**Comment by npruehs** [comment]
Sorry, but with a baby and two jobs, I kinda have to be able to look at these issues "spontaneously" :)

**Comment by LegendsofBarcraft** [comment]
Haha! No worries. I understand ;)
